test/distributed/checkpoint/test_traverse.py

```diff
@@ -5,7 +5,7 @@
 import torch
 
 import torch.distributed.checkpoint._traverse as _traverse
-from torch.distributed.checkpoint.metadata import STATE_DICT_TYPE
+from torch.distributed.checkpoint.metadata import STATE_DICT_TYPE  # noqa: TCH001
```

Comment:
Why not on this one? Here, it seems like it is only used in type annotation context

Reply:
Oh, I just noqa'd all of tests, I'll fix it.